app/components/card.tsx

```diff
@@ -1,3 +1,4 @@
+import { title } from 'process';
```

Comment:
**Suggestion:** The import statement for the `title` variable is not used and should be removed.
```suggestion
import styles from '../styles/card.module.scss';
```